src/SprykerEco/Zed/UnzerApi/Business/UnzerApiFacadeInterface.php

```diff
@@ -0,0 +1,264 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business;
+
+use Generated\Shared\Transfer\UnzerApiRequestTransfer;
+use Generated\Shared\Transfer\UnzerApiResponseTransfer;
+
+interface UnzerApiFacadeInterface
+{
+    /**
+     * Specification:
+     * - Requires `UnzerApiRequestTransfer.unzerKeypair` to be set.
```

Comment:
```suggestion
     * - Requires `UnzerApiRequestTransfer.unzerKeypair.privateKey` to be set.
```

the same for all methods